src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -321,6 +321,42 @@ def _uploadJobDependencies(self, arcJobID, executableFile, inputs, executables):
             self.log.verbose("Input correctly uploaded", fileToSubmit)
         return S_OK()
 
+    def _killJob(self, jobIDList):
```

Comment:
Just to be in line with `renewJobs(self, arcJobList)`:
```suggestion
    def _killJob(self, arcJobList):
```